src/Common/DigipolisPropertiesAware.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\General\Common;
+
+trait DigipolisPropertiesAware
```

Comment:
Documentation on how and when this Trait can (or should) be used, should be added in the README.md file